kafka_consumer/datadog_checks/kafka_consumer/legacy_0_10_2.py

```diff
@@ -120,6 +116,9 @@ def check(self, instance):
             self.log.exception('There was a problem collecting the high watermark offsets')
             return
 
+        # cleanup kafka connections (to all brokers) to avoid leaving stale connections in older kafkas.
+        self._kafka_client.close()
```

Comment:
I'm actually not sure if this is necessary... if the `instance` is long-lived, then the client instance will just be re-used on every run of the check... this is only needed if the `kafka_client` is recreated every single run of the check... @ofek?

Reply:
per slack conversation this will stick around, no need to close